### DISCUSSION TITLE
请问独立部署后端必须使用单独的域名吗

### DISCUSSION THREAD

**Discussion by wwricu** [walinejs/waline#1779] [Q&A] [answered]
把waline用nginx代理到域名的/waline下面，发现管理界面跳转有问题，进入/waline/ui后点击左上角导航，比如说用户管理，会跳转到/waline/waline/user下面加载不出页面，需要手动输入/waline/user才行，这个好像是前端路由，请问nginx应该怎么配置，还是说只能把waline挂到单独域名的/路径下

**Comment by lizheming** [comment, marked as answer]
不是。子路径主要功能都是没问题的，只是后台前端针对子路径没有做适配，介意跳转问题的可以先用域名吧。
